src/NumericalAlgorithms/Integration/GslQuadAdaptive.hpp

```diff
@@ -66,14 +64,11 @@ class GslQuadAdaptiveImpl {
       workspace_;
 };
 
-void check_status_code(const int status_code);
-;
```

Comment:
only this line with the extra `;` 😄 you need the `check_status_code` function

Reply:
Of course. Thank you, I did a dumb.